catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsLoader.java

```diff
@@ -309,17 +315,40 @@ private static boolean anyNull(Object... args) {
   }
 
   private static void saveMetacards(CatalogFramework framework, List<Metacard> metacards) {
-    SECURITY.runAsAdmin(
+    // TODO: This code will be removed once CRUD support is completed
+    /**
+     * Purpose: Circumvents the idea of a system template to make it look like the loaded templates
+     * were created by a std. user. Additional Info: dummyUser is created (temporarily) in
+     * user.properties/user.attributes to demonstrate template creation - This sets the metacard
+     * owner during boot time - This allows the template to be shared when logged in as dummyUser -
+     * Allows this PR to be hero'ed without the need for CRUD functionality
+     */
+    Subject userSubject = SECURITY.getSubject("dummyUser", "dummyPassword");
+    userSubject.execute(
         () -> {
           try {
-            return SECURITY.runWithSubjectOrElevate(
-                () -> framework.create(new CreateRequestImpl(metacards)).getCreatedMetacards());
-          } catch (SecurityServiceException | InvocationTargetException e) {
-            LOGGER.warn(
-                "Can't create metacard for system search template, was unable to elevate privileges",
-                e);
+            framework.create(new CreateRequestImpl(metacards)).getCreatedMetacards();
+          } catch (IngestException e) {
```

Comment:
This can be condensed into one catch block -> `catch (IngestException | SourceUnavailableException)`.  Might want to consider logging the user.